src/main/java/com/mindee/parsing/common/field/LineItemField.java

```diff
@@ -0,0 +1,12 @@
+package com.mindee.parsing.common.field;
+
+import java.util.HashMap;
+
+/**
+ * Represent a single line in a field printable as an rST table.
+ */
+public abstract class LineItemField extends BaseField {
+  public abstract String toTableLine();
+  public abstract String toString();
+  protected abstract HashMap<String, String> printableValues();
```

Comment:
Is printableValues being used by any classes that extend from TaxField etc. or in the same package? If not, we may not need to define it in the abstract class/interface as a protected method. It could be a private method in concrete class.